env_setup/check_vm_settings.yml

```diff
@@ -4,14 +4,18 @@
 # Check VM's settings which only has one network adapter and it
 # should be connected to VM.
 #
+- name: "Get VM's power state"
+  include_tasks: ../common/vm_get_power_state.yml
+
 - name: "Get VM's network adapters info"
```

Comment:
Has got the network adapter info for the existing VM in env_setup.yml, see line 38. We can save the adapter info there and ues it here.